service/src/main/java/org/apache/fineract/cn/notification/service/internal/config/NotificationConfiguration.java

```diff
@@ -103,23 +108,24 @@ public void configurePathMatch(final PathMatchConfigurer configurer) {
 	}
 	
 	@Bean
-	public PooledConnectionFactory jmsFactory() {
+	public PooledConnectionFactory pooledConnectionFactory() {
```

Comment:
Why are you configuring ActiveMQ again? I understand the command library already does that for us and we are using the commandGateway in this service. Also see as you already configured the concurrency: 1-1 in ur application properties file.

Reply:
I will remove the PooledConnectionFactory and jmsTemplate because they are redundant now that you point that out. 

Currently, the concurrency set by the command library is 3-10

But I want to keep the configuration for the JMSListeners. This is because I want to receive only one message for each topic to avoid sending the same notification 3 times.